tests/api_tests/test_participant_summary_api.py

```diff
@@ -4424,3 +4426,59 @@ def test_updated_since_parameter(self):
             unexpected_participant.participantId in response_ids
             for unexpected_participant in unexpected_participant_list
         ))
+
+    def test_remote_identity_verified(self):
+        """ Test to see if a remote ID verification True Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
+        # Set up participant, questionnaire, questionnaire response & send POST request to API
+        participant = self.data_generator.create_database_participant(participantOrigin='vibrent')
+        participant_id = f'P{participant.participantId}'
+        authored = datetime.datetime.now()
+        self.send_consent(participant.participantId, authored=authored)
+        questionnaire_id = self.create_questionnaire("remote_id_verification_questionnaire.json")
+        resource = load_questionnaire_response_json(
+            'remote_id_verification_questionnaire_response.json',
+            questionnaire_id,
+            participant_id
+        )
+        self.send_post("Participant/%s/QuestionnaireResponse" % participant_id, resource)
+        # Get request from API to assert information is accurate
+        summary = self.send_get("Participant/%s/Summary" % participant_id)
+        self.assertEqual(summary['remoteIdVerificationOrigin'], 'vibrent')
+        self.assertEqual(summary['remoteIdVerificationStatus'], True)
+        self.assertEqual(summary['remoteIdVerifiedOn'], '2022-11-30T00:00:00')
+        # Reset Config back to original user client ID
+        user_info['example@example.com']['clientId'] = original_user_client_id
+        config.override_setting(config.USER_INFO, user_info)
+
+    def test_remote_identity_not_verified(self):
+        """ Test to see if a remote ID verification False Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
```

Comment:
Shouldn't need this for the tests, you can do the POST with the default account.